natives.json

```diff
@@ -3870,43 +3870,43 @@
 			"build": "1207"
 		},
 		"0x2E31ACA7477CF00F": {
-			"name": "_0x2E31ACA7477CF00F",
-			"comment": "",
+			"name": "_SET_SOUND_RELATIONSHIP_ON_PED",
+			"comment": "p1: Entity.Relationship\np2:Player, Enemy, Teammate, Neutral",
 			"params": [
 				{
-					"type": "Any",
-					"name": "p0"
+					"type": "Ped",
+					"name": "ped"
 				},
 				{
-					"type": "Any",
+					"type": "const char*",
 					"name": "p1"
 				},
 				{
-					"type": "Any",
+					"type": "const char*",
 					"name": "p2"
 				}
 			],
 			"return_type": "void",
 			"build": "1207"
 		},
 		"0xCE5D0FFE83939AF1": {
-			"name": "_0xCE5D0FFE83939AF1",
+			"name": "_PLAY_SOUND_FROM_ID",
```

Comment:
I don't think this is correct. It's almost identical to `PLAY_SOUND_FRONTEND`, just slightly different parameter usage for `audScriptAudioEntity::PlaySoundFrontend`.
I'd say it's likely something like `PLAY_SOUND_FRONTEND_WITH_SOUND_ID`.